syntax = "proto3";

package sift.ping.v1;

import "google/api/annotations.proto";
import "google/api/field_behavior.proto";
import "protoc-gen-openapiv2/options/annotations.proto";


option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
  info: {title: "Ping Service"}
};

service PingService {
  rpc Ping(PingRequest) returns (PingResponse) {
    option (google.api.http) = {get: "/api/v1/ping"};
  }
}

message PingRequest {}

message PingResponse {
  string response = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
}
